.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rust-analyzer.cargo.cfgs": {
+        "rust_analyzer": null,

```

Comment:
Is it possible to comment in these files? It'd be good to explain what this does.

Reply:
Done